presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java

```diff
@@ -44,14 +46,18 @@
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkState;
+import static com.google.common.base.Verify.verify;
+import static com.google.common.util.concurrent.Futures.allAsList;
 import static io.airlift.concurrent.MoreFutures.getFutureValue;
 import static io.airlift.concurrent.MoreFutures.toListenableFuture;
 import static java.util.Objects.requireNonNull;
 
 public class TableWriterOperator
         implements Operator
 {
-    public static final List<Type> TYPES = ImmutableList.of(BIGINT, VARBINARY);
+    public static final int ROW_COUNT_CHANNEL = 0;
+    public static final int FRAGMENT_CHANNEL = 1;
+    private static final int WRITER_CHANNELS = 2;
```

Comment:
These are the stats channels, right?  Could you call them STATISTICS_CHANNELS

Reply:
This is a total count of WRITER_CHANNELS.